cmd/legacy/legacyparams/params.go

```diff
@@ -32,7 +32,33 @@ var (
 type Params struct {
 	OfflineDisabled bool
 	OfflineSyncMax  int
-	API             api.Params
+	API             API
+}
+
+// API contains api related parameters.
+type API struct {
+	DisableSSLVerify bool
+	Key              string
+	Plugin           string
+	ProxyURL         string
+	SSLCertFilepath  string
+	Timeout          time.Duration
+	URL              string
+}
+
+// String implements fmt.Stringer interface.
+func (p API) String() string {
+	return fmt.Sprintf(
+		"api key: '%s', api url: '%s', plugin: '%s', timeout: %s, disable ssl verify: %t,"+
```

Comment:
Isn't it better to surround strings with double quote instead single quote?

Reply:
Valid question. Actually using `%q` placeholder would be simpler, yes. But the reason I put them is that in the json log output, the message itself will be surrounded with double quotes as well. Consequently we would end up with escaped double quotes, which is harder to read.
```
{"message":"api key: \"00635xxx\", api url: \"https://wakatime.com\", plugin: \"WakaTime/1.23\""}
```
```
```